src/main/java/org/jitsi/videobridge/transform/RtxTransformer.java

```diff
@@ -325,7 +396,7 @@ private boolean encapsulateInRtxAndTransmit(
         if (mediaStream != null)
         {
             rtxPkt.setSSRC((int) rtxSsrc);
-            rtxPkt.setPayloadType(channel.getRtxPayloadType());
+            rtxPkt.setPayloadType(rtxPayloadType);
```

Comment:
Alright, thanks for letting me know. Another smaller issue that can be easily mitigated with the new code is handling multiple codecs (e.g. 100/96 for VP8/RTX and 101/97 for VP9). I have left a note in line 349.